local-proxy/package.json

```diff
@@ -51,5 +52,8 @@
     "lodash": "^4.17.15",
     "tslint": "^5.18.0",
     "typescript": "^3.5.3"
+  },
+  "peerDependencies": {
+    "@binaris/shift-db": "~0.3.0 || ~0.4.0"
```

Comment:
I think this should be `"0.3.1"` - it will be updated by `rush publish` automatically anyway to `"0.4.0"` during publishing.

Reply:
It seems to me that would be incorrect.  `peerDependencies` are supposed to specify supported versions -- they shouldn't move ahead just because a new version is available.  I hope Rush won't advance versions like this.

Reply:
Rush updates peerDependencies, so a `rush publish -a` will make the change, but keep the `~`.
```
-    "@binaris/shift-db": "~0.3.0 || ~0.4.0"
+    "@binaris/shift-db": "~0.4.0"
```

This would allow us to release versions of `@binaris/shift-db` 0.4.1 0.4.2 0.4.3... When we'll make a minor change in `@binaris/shift-db` we'll probably need to release a new version of `@binaris/shift-local-proxy`.

I don't really want to support `~` in versions, I would like that it's possible to have exactly one combination of packages and not various combinations, but it probably needs to be solved on a different level.

Reply:
Not sure how that will work with vendoring.  Specifically, suppose I want to publish some ShiftJS-capable component.  I depend on some version of `shift-db`, but it will _not_ be the one that is necessarily installed in the app which uses my component.
Anyway, I want to defer fixing this to "later".